### DISCUSSION TITLE
Deepkit X

### DISCUSSION THREAD

**Discussion by marcj** [decs/typeschema#31] [General]
looks a bit weird to me that at the column Infer is an X for deepkit. It might appear like deepkit doesn't support it, when in fact it doesn't need that workaround. it actually operates on types, not on JS objects, so it is actually good to not have a Infer utility, since these utility are usually crazy slow in type checking performance, especially in bigger code bases.

What do you think about replacing the icon with something that is not red and does not indicate "bad". maybe also a green checkmark or something completely different that highlight the fact that deepkit does not need that utility to get the same functionality.

**Comment by decs** [comment]
hey, @marcj! I appreciate the feedback!

indeed the red X can be perceived as negative, and it's not the message we're trying to pass. I've replaced it with the 🧐 emoji, to indicate that we're looking for ways to support that functionality. what do you think?
